plugins/chat/spec/services/chat/create_category_channel_spec.rb

```diff
@@ -16,6 +16,14 @@
     let(:guardian) { Guardian.new(current_user) }
     let(:params) { { guardian: guardian, category_id: category_id, name: "cool channel" } }
 
+    context "when public channels are disabled" do
+      fab!(:current_user) { Fabricate(:user) }
+
+      before { SiteSetting.enable_public_channels = false }
+
+      it { is_expected.to fail_a_policy(:public_channels_enabled) }
```

Comment:
I am guessing this is failing ATM because of the typo

Reply:
what would I do without tests :D